lib/generate-report.js

```diff
@@ -178,6 +180,9 @@ function generateReport(options) {
       feature.id = `${uuid()}.${feature.id}`.replace(/[^a-zA-Z0-9-_]/g, '-');
       feature.app = 0;
       feature.browser = 0;
+      
+      var uriSegments = feature.uri.split('\\');
+      feature.folder = uriSegments[uriSegments.length - 2] || '-';
```

Comment:
I thought about the option. Hence included a parameter to configurable. By, default folder name display is set to false. Hence, those who have multiple/complex folder structure, They can continue to have the same reporting structure.
As stated in the above example, an user having accordion.feature will be listed in different rows with their corresponding test results. Looking at that report is confusing as I need to remember the number of test cases etc while doing a walkthrough to business.. And this creates a  dependency of QA engineer to explain the report every time and hence I wanted to group them. 

I understand, this new option is a baby step. But it will ease the user in grouping the list of features from a specific folder to quickly walk through the report to business. In my case, they are individual module, hence easily understandable by looking at the report. No tester input required in clarifying which feature corresponds to what.